modules/pam_access/pam_access.c

```diff
@@ -623,6 +623,8 @@ from_match (pam_handle_t *pamh UNUSED, char *tok, struct login_info *item)
     } else if (item->from_remote_host == 0) {	/* local: no PAM_RHOSTS */
 	if (strcasecmp(tok, "LOCAL") == 0)
 	    return (YES);
+    } else if (strcasecmp(tok, "LOCAL") == 0) {
+        return NO;
```

Comment:
This can be rewritten in a more logical way:
```
} else if (strcasecmp(tok, "LOCAL") == 0) {
   if (item->from_remote_host == 0)
       return YES;
   return NO;
} else if ....
```

Reply:
That is not the same logic. With your suggestion, local accesses would fall through, when `LOCAL` is not configured.

Reply:
IMHO while this is not the same logic, it has the same outcome:
from_match() only returns YES if
item->from_remote_host == 0 AND strcasecmp(tok, "LOCAL") == 0
for both patches.
One could even leave out the "return NO;" because that's the default at the end of the function.

Reply:
You're right. But I still prefer doing it differently so that the comparison of tok with LOCAL is not done twice. My alternate PR: #345

Reply:
Okay, that clean-up makes it more readable. I just added some minor comments.